src/App.js

```diff
@@ -71,11 +84,33 @@ function App() {
       console.log(error.message);
     }
   };
+  const removeTodoList = async (id) => {
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default/${id}`;
+    const options = {
+      method: "DELETE",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+        "Content-Type": "application/json",
+      },
+    };
+    try {
+      const response = await fetch(url, options);
+      if (!response.ok) {
+        const message = `Error has occured: ${response.status}`;
+        throw new Error(message);
+      }
+      const data = await response.json();
+      console.log(data);
```

Comment:
Nitpick:
You should use the `data` returned the response to filter your `todoList`. You be able to combine this `removeTodoList` function with the `removeTodo` function:
```js
  const removeTodo = async (id) => {
    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default/${id}`;

    const options = {
      method: "DELETE",
      headers: {
        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
        "Content-Type": "application/json",
      },
    };

    try {
      const response = await fetch(url, options);

      if (!response.ok) {
        const message = `Error has occured: ${response.status}`;
        throw new Error(message);
      }
      
      // deleting a todo from Airtable will respond with the deleted todo's id within the response data
      const deletedTodo = await response.json();

      // using the response data, you can filter the todoList using the deleted todo's id
      setTodoList(prevTodoList => prevTodoList.filter((item) => item.id !== deletedTodo.id));
    } catch (error) {
      console.log(error.message);
    }
  };
```